.travis.yml

```diff
@@ -11,3 +11,7 @@ rvm:
   - 1.9.3
   - jruby-19mode
   - rbx-2.2.10
+
+matrix:
+  allow_failures:
+    - rvm: rbx-2.2.10
```

Comment:
Both. The first one says that we use `rbx-2` and the second that we don't care ;)